docs/graphql/manual/getting-started/hasura-cloud.rst

```diff
@@ -0,0 +1,67 @@
+.. meta::
+   :description: Hasura Pro getting started
+   :keywords: hasura, docs, cloud, signup
+
+.. _getting_started_hasura_cloud:
+
+Quickstart with Hasura Cloud
+============================
+
```

Comment:
This page seems to be a duplicate of https://hasura.io/docs/cloud/1.0/manual/getting-started/index.html. Was this intentional? Ideally, we should avoid duplicate content since it will be harder to maintain. Maybe a short description and a link instead?

Reply:
@sezgi I agree that we should avoid duplicate content and was thinking the same. My though process was that we want people to use Hasura Cloud and that at least a basic guide should be there, and then link to everything else in the cloud docs. What's your opinion on this @rikinsk?

Reply:
P.S.: I also made it to be more like a deployment guide as the others are in our OSS docs, with steps etc.

Reply:
Ah I see -- like a teaser :) We can show the project dashboard image and a short description as a teaser, and link to the cloud docs from there. That way we get their attention & also avoid the duplicate content problem. How does that sound?

Reply:
Better yet, can the sidebar link go straight to https://hasura.io/docs/cloud/1.0/manual/getting-started/index.html ? It's essentially the same page.

Reply:
Done :)